src/helpers/manage-merge-queue.ts

```diff
@@ -42,6 +43,10 @@ export const manageMergeQueue = async ({ login, slack_webhook_url }: ManageMerge
     core.info('This PR is not in the merge queue.');
     return removePrFromQueue(pullRequest);
   }
+  if (!pullRequest.labels.find(label => label.name === CORE_APPROVED_PR_LABEL)) {
+    core.info('This PR is not core approved.');
```

Comment:
I wonder if leaving a comment on the PR using [createPrComment](https://github.com/ExpediaGroup/github-helpers/blob/main/src/helpers/create-pr-comment.ts#L52) would be better than just logging in the job

Reply:
That makes sense! Will do that.